lib/themes/dosomething/paraneue_dosomething/includes/helpers.inc

```diff
@@ -346,7 +346,7 @@ function paraneue_dosomething_build_reportbacks_gallery_collection ($reportbacks
       $reportbacks['prefetched'] = 0;
     }
     else {
-      return FALSE;
+      return NULL;
```

Comment:
@aaronschachter I made a quick change here cause returning `FALSE` apparently is no good if using `isset()` in template cause it sees it as the variable having a value (which makes sense). So I changed it to return `NULL` so the conditional in template works correctly.